data/ecosystem.yaml

```diff
@@ -86,8 +86,8 @@ navigator:
   docs:
     home: "https://ansible.readthedocs.io/projects/navigator/"
 pylibssh:
```

Comment:
@oraNod I missed originally there's this YAML key that should probably be updated to `ansible-pylibssh` too.